packages/utils/telemetry-utils/src/staging.ts

```diff
@@ -0,0 +1,43 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ILoggingError } from "@fluidframework/common-definitions";
+
+// ///////////////////////////////////////////////////////////////////////// //
+//        THIS CODE TO BE MOVED TO COMMON-DEFINITIONS AND COMMON-UTILS       //
+// ///////////////////////////////////////////////////////////////////////// //
+
+export interface IFluidErrorBase extends Readonly<Error> {
```

Comment:
should this also extend Record<string, unknown>, or ITelemetryProperties?

Reply:
No. I removed the type constraint on `mixinTelemetryProps` anyway. And I want to keep logging decoupled from the base error type.